README.md

```diff
@@ -129,7 +173,7 @@ The connector will go through the left_joins in order, looking them up and mergi
 
 #### Selecting Whole Models Instead of Fields
 
-Instead of specifying the precise fields you want, you can instead include the entire joined model in your model.
+Instead of specifying the precise fields you want, you can instead include the entire joined model in your model. The property "name" should *not* be used.
```

Comment:
"'should' implies 'can'" - well, it 'can' be used :) you just end up with different functionality.

I can make it positive, but "create a field in your composite connector that has the type "Object" and the name of the source model only" is incorrect i think. I'm sure there are other properties which can be used alongside this. Can't think of what they are though. Default maybe? The main thing is that if you specify "name" you won't get this, so don't do it. Any alternative ideas?